### DISCUSSION TITLE
L2 construct for MWAA (Ariflow) environment

### DISCUSSION THREAD

**Discussion by shmuelgutman** [aws/aws-cdk#24070] [General]
Hi all,

I'm curious how many of you are interested in an L2 construct for the MWAA environment, which could include things like:
- MWAA-Environment
- Execution role
- Execution policies 
- Security-groups
And maybe more resources.

I'm currently working on this for my own used.

**Comment by indrora** [comment]
We actively encourage such things to be submitted to the AWS CDK Construct Library at https://constructs.dev

**Comment by Jacco** [comment]
The roadmap of cdk (https://github.com/aws/aws-cdk/blob/main/ROADMAP.md) clearly states in objective 1:

"Our vision is to provide L2 constructs for all AWS services, and timely maintain and enhance existing L2 constructs as their underlying services evolve."

Apart from that construct hub already has: https://github.com/hupe1980/cdk-mwaa

I am currently working on an L2 construct for MWAA https://github.com/aws/aws-cdk/issues/30645
